app/components/WoWResults/FullScan/MobileDeltaTable.tsx

```
@@ -0,0 +1,193 @@
+import { ChevronDownIcon, ChevronUpIcon } from '@heroicons/react/outline'
+import { Title } from '~/components/Common'
+import Label from '~/components/form/Label'
+import Modal from '~/components/form/Modal'
+import type { ColumnList } from '~/components/types'
+import DebouncedInput from '~/components/Common/DebouncedInput'
+
+const parseToLocaleString = (value: any) => {
+  if (typeof value === 'number') {
+    return value.toLocaleString()
+  }
+  if (typeof value === 'string' && !isNaN(parseFloat(value))) {
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)